rest/src/main/scala/io/udash/rest/openapi/OpenApiMetadata.scala

```diff
@@ -52,6 +52,16 @@ final case class OpenApiMetadata[T](
 ) {
   val httpMethods: List[OpenApiOperation[_]] = (gets: List[OpenApiOperation[_]]) ++ customBodyMethods ++ bodyMethods
 
+  // collect all tags
+  private lazy val openApiTags: List[Tag] = {
```

Comment:
Will this ever not be evaluated (assuming `OpenApiMetadata` is created) to benefit from laziness? Only when the metadata is not accessed at all? How much computation are we actually saving?

Reply:
It will be evaluated when `OpenApi` instance is created (io.udash.rest.openapi.OpenApiMetadata#openapi is called). I'm not sure if lazy val is necessary here, but on the other hand other Open API classes like Operations (in io.udash.rest.openapi.OpenApiMetadata#operations) are also created on demand when `openapi` method is called